samplers/parser.go

```diff
@@ -75,6 +76,35 @@ func ConvertMetrics(m *protocol.Message) ([]UDPMetric, error) {
 	return metrics, nil
 }
 
+// ConvertIndicatorMetrics takes a span that may be an "indicator"
+// span and returns metrics that can be determined from that
+// span. Currently, it converts the span to a timer metric for the
+// duration of the span.
+func ConvertIndicatorMetrics(span *ssf.SSFSpan, timerName string) (metrics []UDPMetric, err error) {
+	if !span.Indicator || timerName == "" {
+		// No-op if this isn't an indicator span
+		return
+	}
+
+	end := time.Unix(span.EndTimestamp/1e9, span.EndTimestamp%1e9)
+	start := time.Unix(span.StartTimestamp/1e9, span.StartTimestamp%1e9)
+	ssfTimer := &ssf.SSFSample{
+		Metric: ssf.SSFSample_HISTOGRAM,
+		Name:   timerName,
+		Value:  float32(end.Sub(start) * time.Millisecond),
+		Tags: map[string]string{
+			"name":    span.Name,
+			"service": span.Service,
```

Comment:
One last thing - can we also add `span.Error` (a bool) as a tag?

Reply:
Great idea! Doing.